### PR TITLE
Upgrade powsybl

### PR DESCRIPTION
**Pull request by EtienneLt**

### CODE REVIEW

File: k8s/resources/study/config/voltage-init-server-config.yml [hunk 1]
```diff
@@ -3,4 +3,7 @@ network:
 
 # Nb of concurrent processes - should be in-sync with voltageinit.run.voltageinitGroup queue concurrency
 computation-local:
-  available-core: 2
\ No newline at end of file
+  available-core: 2
+
```
Comment (etiennehomer) [inline k8s/resources/study/config/voltage-init-server-config.yml:7]: // TODO: to remove when upgrade to powsybl-optimizer v0.9.0 is done

File: docker-compose/study/docker-compose.override.yml [hunk 2]
```diff
@@ -509,7 +509,7 @@ services:
     ports:
       - 5038:80
     volumes:
-      - $PWD/../../k8s/resources/study/config/voltage-init-server-application.yml:/config/specific/application.yml:Z
+      - $PWD/../../k8s/resources/study/config/voltage-init-server-config.yml:/config/specific/application.yml:Z
```
Comment (etiennehomer) [inline docker-compose/study/docker-compose.override.yml:512]: to revert ?

**Review by etiennehomer — CHANGES_REQUESTED** [review]

File: docker-compose/study/docker-compose.override.yml (hunk 2 above)
Comment (EtienneLt) [reply docker-compose/study/docker-compose.override.yml:512]: yes

File: k8s/resources/study/config/voltage-init-server-config.yml (hunk 1 above)
Comment (EtienneLt) [reply k8s/resources/study/config/voltage-init-server-config.yml:7]: yes

**Review by etiennehomer — APPROVED** [review]

**Pull request merged**